Configuration-Interaction/CI_DL.py

```diff
@@ -117,20 +112,14 @@
 for CI_ITER in range(max_guess - 1):
 
     # Subspace Matrix, Gij = < bi | H | bj >
-    #for i in range(num_vecs - num_eig, num_vecs):
     for i in range(0, num_vecs):
         # Build sigma for each b
         ciwfn.sigma(cvecs, svecs, i, i)
```

Comment:
cvecs and svecs are both initialized with `max_guess` number of vectors, but the program exits when I put in a `svecs.read(i,0)`, even on the first iteration of `i` which gives the correct vector:

    PSIO_ERROR: Can't find TOC Entry buffer_ 0
    PSIO_ERROR: unit = 201, errval = 13
    PSIO_ERROR: 13 (no such TOC entry)
    Traceback (most recent call last):
      File "CI_DL.py", line 118, in <module>
        svecs.read(i,0)
    RuntimeError: 
    Fatal Error: PSIO Error